python/pyfunc-server/docker/base.Dockerfile

```diff
@@ -12,14 +12,17 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM continuumio/miniconda3
+FROM condaforge/miniforge3
+
+ARG PYTHON_VERSION
 
 RUN wget -qO- https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-367.0.0-linux-x86_64.tar.gz | tar xzf -
```

Comment:
batch-repdictor base image uses `ARG GCLOUD_VERSION=405.0.1`. can we also update pyfunc to use 405.0.1?